documentation/blog/2025-04-17-goose-goes-to-NY/index.mdx

```diff
@@ -0,0 +1,123 @@
+---
+title: "Codename Goose Goes to New York"
+description: "Goose lands in NYC for its second ever community meetup"
+authors: 
+    - ebony
+---
+
+import ImageCarousel from '@site/src/components/ImageCarousel';
+import swag from './swag2.jpg';
+import swag1 from './swag1.jpg';
+import swag2 from './swag.jpg';
+import swag3 from './swag3.jpg';
+import focus from './focus.jpg';
+import focus2 from './focus2.jpg';
+import focus3 from './focus3.jpg';
+import focus4 from './focus5.jpg';
+import focus5 from './focus4.jpg';
+import speaker from './speaker.jpg';
+import speaker1 from './speaker1.jpg';
+import fun from './fun.jpg';
+import fun1 from './fun1.jpg';
+
+
+![blog cover](cover.png)
+
+# Codename Goose Goes to New York 🗽
+
+We brought Goose to New York and NYC was one for the books.
```

Comment:
```suggestion
We brought Goose to New York City, and the event was one for the books.
```

feel free to switch it up. I'm just thinking the New York then NYC seems repetitive.